src/esp/physics/PhysicsManager.cpp

```diff
@@ -341,6 +341,30 @@ Magnum::Quaternion PhysicsManager::getRotation(const int physObjectID) const {
   return existingObjects_.at(physObjectID)->rotation();
 }
 
+void PhysicsManager::setLinearVelocity(const int physObjectID,
+                                       const Magnum::Vector3& lin_vel) {
```

Comment:
Based on the C++ convention, we use camel case for variables.
Suggest to change the variables in this PR.